mobilecoind/clients/python/blockchain_explorer.py

```diff
@@ -0,0 +1,39 @@
+from mob_client import mob_client
+from flask import Flask
+from flask import render_template
+from sys import getsizeof
+
+mob_client = mob_client('localhost:4444', False)
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    num_blocks, num_transactions = mob_client.get_ledger_info()
+
+    blocks = []
+    for i in range(num_blocks - 1, max(num_blocks - 100, -1), -1):
+        _key_image_count, txo_count = mob_client.get_block_info(i)
+        block_hash, signature, txos, key_images = mob_client.get_block_details(i)
+        size_of_block = sum([getsizeof(x) for x in [block_hash, signature, txos, key_images]]) * .001
```

Comment:
This is not correct - I am not sure what size we're trying to calculate, but the Block has an ID, version field, root element, etc that are unaccounted for. I think the size column can be removed.

Reply:
We need the size column as per the PRD - I will play around with getting a closer approximation, and will indicate it's an approximation with `~kB`.

Reply:
I'd actually check in if this is super important or was a bit whimsical...

Reply:
Yep - checked with @rjwalters. I think if I use getsizeof(block), where block is the external.Block, maybe that's an ok approximation? What do you think, @eranrund?

Reply:
external.Block + the associated txouts and key images. Of course thats just the size of the raw data, but blocks are moved everywhere as protobufs so I'm not exactly sure what the goal here is 🤷‍♂️

Reply:
if there is data in the blockchain that is not displayed (i.e. ID, version, root element) I'd prefer to actually add this to the explorer

Reply:
This would change what is surfaced by `GetBlockDetails`

Reply:
Yep - changing GetBlockDetails to GetBlock now, and we can provide visibility into the contents in the explorer from there.